backend/coins/btc/account.go

```diff
@@ -69,14 +75,12 @@ type Account struct {
 	db                      transactions.DBInterface
 	forceGapLimits          *types.GapLimits
 	getSigningConfiguration func() (*signing.Configuration, error)
-	signingConfiguration    *signing.Configuration
 	keystores               *keystore.Keystores
 	getNotifier             func(*signing.Configuration) accounts.Notifier
 	notifier                accounts.Notifier
 	blockchain              blockchain.Interface
 
-	receiveAddresses AddressChain
-	changeAddresses  AddressChain
+	subaccounts []subaccount
```

Comment:
Would a map instead of a slice make things easier? Keyed by ScriptType.

Reply:
Looking at my branch with the rest of the commits, a slice seems to work fine. There is only one lookup by script type needed, but that is in the bb02 keystore, so at best I might convert the signing configs in the tx proposal to a map, but not here.

if we need lookups in the future, changing it to a map should be very simple (iterating over them works the same way).

Reply:
:+1: